apps/src/applab/commands.js

```diff
@@ -1795,6 +1799,47 @@ applabCommands.drawChartFromRecords = function (opts) {
   ).then(onSuccess, onError);
 };
 
+applabCommands.pinMode = function (opts) {
+  apiValidateType(opts, 'pinMode', 'pin', opts.pin, 'pinid');
+  apiValidateType(opts, 'pinMode', 'mode', opts.mode, 'string');
+
+  var modeStringToConstant = {
+    'input': 0,
+    'output': 1,
+    'analog': 2,
+    'pwm': 3,
+    'servo': 4
+  };
```

Comment:
these don't need quotes around them. you could also make this a const instead of a var now :)